routes/error-tracker.js

```diff
@@ -209,11 +203,17 @@ function getHandler(req, res) {
       },
     },
   };
-  // get authentication context for logging
-  const loggingClient = logging({
-    projectId: appEngineProjectId,
-  });
-  const log = loggingClient.log(logName);
+  if (params.debug === '1') {
+    res.set('Content-Type', 'application/json; charset=ISO-8859-1');
+    res.status(statusCodes.OK).send(
+        JSON.stringify({
```

Comment:
Do you need to stringify this? It looks like sending an object works.

http://expressjs.com/en/api.html#res.send